.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
   - '0.10'
+  - '0.12'
```

Comment:
yeah, I want all the projects to be tested on 0.10, 0.12 and 4

I'm not actually sure why travis didn't run for this PR or some of the others.  I don't see any options related to that.